code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -942,7 +942,11 @@ GLOBAL_LIST_EMPTY(aide_list)
 		if(3)
 			new /obj/item/dragon_egg(src)
 		if(4)
-			new /obj/item/dragons_blood(src)
+			loot = rand(1,4)
+			if(loot == 4)	//Still same chance but now you know if you're turning into a lizard (ew)
+				new /obj/item/dragons_blood/refined(src)
+			else
+				new /obj/item/dragons_blood(src)
```

Comment:
just use prob(25) with an else if you're only caring about a single value of rand()